core/modules/locale/tests/locale.test

```diff
@@ -2808,3 +2808,46 @@ class LocaleLanguageNegotiationInfoFunctionalTest extends BackdropWebTestCase {
     $this->backdropGet('admin/config/regional/language/detection');
   }
 }
+
+/**
+ * Tests locale translation safe string handling.
+ */
+class LocaleStringIsSafeTest extends BackdropWebTestCase {
+  public static function getInfo() {
+    return array(
+      'name' => 'Test if a string is safe',
+      'description' => 'Tests locale translation safe string handling.',
+      'group' => 'Locale',
+    );
+  }
```

Comment:
This test isn't going to be called because Backdrop uses *.tests.info files to identify test classes rather than `getInfo()` methods. Could you put this in `locale.tests.info` and remove this method?